src/utils/components/carousel/index.ts

```diff
@@ -0,0 +1,19 @@
+import {calculateItemsPerSlideProps} from "@src/utils/components/carousel/types.ts";
+
+export const calculateItemsPerSlide = ({parentWidth, minItemWidth, maxItemWidth} : calculateItemsPerSlideProps) => {
```

Comment:
Could we add a few documentation to have a detail about how it works? And what it does?